scripts/flash_attention.py

```diff
@@ -3,22 +3,49 @@
 import argparse
 
 import torch
+import triton
 
-from triton_kernels_benchmark.flash_attention_benchmark import _attention
+from triton_kernels_benchmark.flash_attention_benchmark import _attention, tune_attn_fwd
 
 
 def get_options():
     """Gather CL options."""
     parser = argparse.ArgumentParser(prog='flash-attention', description='Run Intel XPU Flash-Attention implementation')
-    parser.add_argument('-Z', type=int, required=True, help='Batch size')
-    parser.add_argument('-H', type=int, required=True, help='Head count')
-    parser.add_argument('-N-CTX', type=int, required=True, help='Sequence length')
-    parser.add_argument('-D-HEAD', type=int, required=True, help='Embedding dimension')
-    parser.add_argument('-causal', action='store_true', help='Run causal attention')
-    parser.add_argument('-backward', action='store_true', help='Run backward attention')
+
+    model = parser.add_argument_group(title='Model description',
+                                      description='Options setting different model metaparameters')
+    model.add_argument('-Z', type=int, required=True, help='Batch size')
+    model.add_argument('-H', type=int, required=True, help='Head count')
+    model.add_argument('-N-CTX', type=int, required=True, help='Sequence length')
+    model.add_argument('-D-HEAD', type=int, required=True, help='Embedding dimension')
+    model.add_argument('-causal', action='store_true', help='Run causal attention')
+    model.add_argument('-backward', action='store_true', help='Run backward attention')
+
+    config = parser.add_argument_group(title='Tuning configuration',
+                                       description='Options setting different tuning parameters')
+    config.add_argument('-BLOCK-M', action='extend', nargs='+', type=int, help='Sizes of m')
+    config.add_argument('-BLOCK-N', action='extend', nargs='+', type=int, help='Size of N')
```

Comment:
can be the other way around, just for consistency.
```suggestion
    config.add_argument('-BLOCK-N', action='extend', nargs='+', type=int, help='Sizes of n')
```